tasks/licenses_csv_report.js

```diff
@@ -0,0 +1,72 @@
+import { writeFileSync } from 'fs';
+import { getInstalledPackages } from './lib';
+
+import {
+  isNull,
+  isUndefined
+} from 'lodash';
+
+const allDoubleQuoteRE = /"/g;
+
+function escapeValue(value) {
+  if (isNull(value)) {
+    return;
+  }
+
+  return `"${value.replace(allDoubleQuoteRE, '""')}"`;
+}
+
+function formatCsvValues(fields, values) {
+  return fields.map((field) => {
+    const value = values[field];
+
+    if (isNull(value) || isUndefined(value)) {
+      return null;
+    }
+
+    return value.toString();
+  })
+    .map(escapeValue)
+    .join(',');
+}
+
+export default function licensesCSVReport(grunt) {
+  grunt.registerTask('licenses:csv_report', 'Report of 3rd party dependencies', async function () {
+    const fields = ['name', 'version', 'revision', 'url', 'license', 'copyright'];
```

Comment:
'revision' is in the list of fields, but it's not in any of the data, so there's always an empty field in the CSV

Reply:
Yeah, it's a requirement for the output format.

Reply:
@mgreau, can we omit the column altogether?

Reply:
Gotcha, didn't realize having the column there was part of the requirements. Nevermind then!

Reply:
@tylerhannan @kobelb if you know that the `revision` field will never be valued you can remove the column, same for `copyright` column as it is an optional column in this first version of the script. And yes the first line of the file is the header line (columns names).

Reply:
Great, updated: https://docs.google.com/a/elastic.co/spreadsheets/d/1_7HgSd6p9-u6My62PR7K90SQ-wn1oI0_x_IKUDH-ufE/edit?usp=sharing